app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/InsuranceDestination.kt

```diff
@@ -231,35 +231,37 @@ private fun InsuranceScreenContent(
       )
     }
     Spacer(Modifier.height(16.dp))
-    Column(verticalArrangement = Arrangement.spacedBy(24.dp)) {
+    Column(verticalArrangement = Arrangement.spacedBy(16.dp)) {
       if (uiState.isLoading) {
         InsuranceCardPlaceholder(
           imageLoader = imageLoader,
           modifier = Modifier.padding(horizontal = 16.dp),
         )
         CrossSellItemPlaceholder(Modifier.padding(horizontal = 16.dp))
       } else {
-        ContractsSection(
-          imageLoader = imageLoader,
-          onInsuranceCardClick = onInsuranceCardClick,
-          contracts = uiState.contracts,
-        )
-        if (uiState.travelAddonBannerInfo != null) {
-          TravelAddonBanner(
-            travelAddonBannerInfo = uiState.travelAddonBannerInfo,
-            launchAddonPurchaseFlow = {
-              onNavigateToAddonPurchaseFlow(uiState.travelAddonBannerInfo.eligibleInsurancesIds)
-            },
-            modifier = Modifier
-              .fillMaxWidth()
-              .padding(horizontal = 16.dp),
-          )
-        }
-        if (uiState.shouldSuggestMovingFlow) {
-          MovingFlowSuggestionSection(
-            onNavigateToMovingFlow = onNavigateToMovingFlow,
-            modifier = Modifier.padding(horizontal = 16.dp),
+        Column(verticalArrangement = Arrangement.spacedBy(8.dp)) {
```

Comment:
Wrapped all cards in a column with 8.dp spacing between them so the outer column spacing of 16 applies to everything else, and not what is inside this column itself which has its own internall spacing